py/shared/emails/plumbing.py

```diff
@@ -343,7 +343,7 @@ async def send_emails(
         )
 
     @concurrent('low')  # noqa: C901 (ignore complexity)
-    async def record_email_event(self, raw_message: str):
+    async def record_email_event(self, raw_message: str):  # noqa: C901
```

Comment:
I doubt this is necessary since the function hasn't changed.

Reply:
It is. The tests don't run  without it:
`py/shared/emails/plumbing.py:346:5: C901 'BaseEmailActor.record_email_event' is too complex (13)`

Reply:
probably because you're using a different version of python or flake8